src/client/components/ConsentsSubHeader.tsx

```diff
@@ -66,15 +66,29 @@ const circle = `
 `;
 
 const li = (numPages: number) => css`
-  ${textSans.small()}
+  ${textSans.xxsmall()}
+  ${from.mobileMedium} {
+    ${textSans.xsmall()}
+  }
+  ${from.phablet} {
+    ${textSans.small()}
+  }
   position: relative;
   width: ${100 / numPages}%;
   display: flex;
   flex-direction: column;
-  justify-content: flex-end;
+  justify-content: flex-start;
+  padding-top: ${space[6] + space[2]}px;
```

Comment:
just wondering if there's single space value for this instead of having to add them together?

Reply:
Unfortunately not: http://theguardian.design/2a1e5182b/p/449bd5-space

Reply:
hm, if we have to make a space value that doesn't exist on the scale is that circumventing the design system? not sure how strict we want to be at sticking to their scale. I realise this is a tiny issue but thought worth bringing up :)

Reply:
Yeah, in general we try to stick to the guidelines, in this case it was a bit different as rather than padding between elements, we had to pad in the element to push the text down from the step counter line:

![Inkedwelcome_LI](https://user-images.githubusercontent.com/13315440/130030355-c87cf0e1-fca6-4e28-b34d-d7c5ea31ad4c.jpg)